src/Presentation/Presenters/DonationConfirmationHtmlPresenter.php

```diff
@@ -19,13 +19,14 @@ public function __construct( TwigTemplate $template ) {
 		$this->template = $template;
 	}
 
-	public function present( Donation $donation, string $updateToken ): string {
-		return $this->template->render( $this->getConfirmationPageArguments( $donation, $updateToken ) );
+	public function present( Donation $donation, string $updateToken, string $templateName = '' ): string {
```

Comment:
Ok. Then how about just making the $templateName here required? And perhaps rename it somehow to indicate this is not the name of the main template.